scss/civicrm/common/_forms.scss

```diff
@@ -701,6 +682,22 @@
       color: $crm-white;
     }
   }
+
+  //Page specific
```

Comment:
shouldn't we have a partial just for this pages then?

Reply:
No, we don't have any directory for placeholders and there are many placeholders scattered around different styles.

Reply:
Yes, that makes sense, I never had look on here. Caught this due to linter changing orders